### PR TITLE
Updated fast gettext version req to 1.1 (minor digit)

### PR DESCRIPTION
**Pull request by majioa**

### CODE REVIEW

**Comment by majioa** [comment]
Yes, patch is trivial, with no regression intercepted

**Comment by Magisus** [comment]
That version of fast_gettext drops Ruby 1.9 support (hence our explicit pin to 1.1.x), which is still supported in Puppet < 6. We probably can't update this without branching this gem.

**Comment by majioa** [comment]
@Magisus are they really dropped the 1.9 version support?

**Comment by Magisus** [comment]
Yep, see https://rubygems.org/gems/fast_gettext. They dropped it back in 1.2 (without even a release note). I actually had to work with the maintainer to get him to branch the gem for us and backport some fixes we needed to the Ruby 1.9-supporting version.

**Comment by Magisus** [comment]
Is there some feature in particular you were looking for? If so it might be possible to get it backported. Or was this just maintenance?

**Comment by majioa** [comment]
@Magisus i'm working for the Altlinux, and we build the packages, particularly Puppet, which has *fast_gettext* in dependencies, but we have it built 1.7 version, and we have some other packages (exp. foreman) that depends on this version of 1.7 gem. So our task is to to keep puppet and foreman working together in a single default bundle space. That is why I propose the changes.

File: gettext-setup.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 1.9.2'
 
-  spec.add_dependency 'fast_gettext', '~> 1.1.0'
+  spec.add_dependency 'fast_gettext', '~> 1.7'
```
Comment (smortex) [inline gettext-setup.gemspec]: I guess `~> 1.1` would be enough, allowing both 1.1.x and 1.7+.

I was just about to propose this change, with basically the same idea behind my head regarding the FreeBSD ports of Puppet, see:
https://github.com/puppetlabs/puppet/pull/7406

Comment (Magisus) [reply gettext-setup.gemspec]: Yes I think `~> 1.1` would work for us everywhere.

Comment (majioa) [reply gettext-setup.gemspec]: Well, may be yes

Comment (majioa) [reply gettext-setup.gemspec]: @nigelkersten what is next?

Comment (smortex) [reply gettext-setup.gemspec]: @majioa you can amend this commit to depend on `~> 1.1` and then rebase your branch on top of master to get rid of the conflicts.  You will then need to force push and the PR should be in good shape.

Comment (majioa) [reply gettext-setup.gemspec]: updated

**Pull request merged**